package.json

```
@@ -6,16 +6,16 @@
   "scripts": {
     "build:app": "turbo run build --filter=next-auth-app --include-dependencies",
     "build": "turbo run build --filter=next-auth --filter=@next-auth/* --no-deps",
-    "lint": "turbo run lint --filter=!next-auth-docs",
-    "lint:fix": "turbo run lint -- --fix",
-    "test": "turbo run test --concurrency=1 --filter=!@next-auth/pouchdb-adapter --filter=!next-auth-* --filter=[HEAD^1]",
-    "setup": "turbo run setup",
+    "clean": "turbo run clean --no-cache",
+    "dev:app": "turbo run dev --parallel --filter=next-auth-app",
```

> I'm not sure why this is necessary though:
> 
> https://github.com/nextauthjs/next-auth/blob/18728558cc95cee8559ae4abd47bf42b635f301b/turbo.json#L11-L26
> 
> I thought this should be enough:
> 
> ```json
> "dev": {
>   "cache": false,
>   "dependsOn": ["^build"]
> },
> ```
> 
> But I have to explicitly add the build of the adapters, and define that each adapter `dependsOn` `next-auth#build` (because of the types)

Try this: 
```suggestion
    "dev:app": "turbo run dev --filter=next-auth-app...",
```

Two things:
- Remove `--parallel` because `turborepo` ignores the dependency graph with this option. Not sure why it was added though, maybe because we tried to run two packages at the same time then 😁 
- Filter dependent packages with `...` as described [here](https://turborepo.org/docs/core-concepts/filtering#include-dependencies-of-matched-packages)